src/main/driver_unified.cpp

```diff
@@ -146,14 +146,19 @@ int runCvc4(int argc, char* argv[], Options& opts) {
   }
 
   // Auto-detect input language by filename extension
-  const char* filename = inputFromStdin ? "<stdin>" : filenames[0].c_str();
+  std::string filenameStr("<stdin>");
+  if (!inputFromStdin) {
+    // TODO: use std::move() when switching to c++11
+    filenameStr.swap(filenames[0]);
```

Comment:
The vector `filenames` is not used anywhere else, so I thought that we could avoid copying the string and instead just `std::move` it to `filenameStr` (the `string::swap` being the pre-c++11 alternative to `std::move`). `filenames` and `filenameStr` are both stack allocated in `runCvc4()` and owned by it.